### DISCUSSION TITLE
AddCertificate_Orchestrator fails for ASP.NET application

### DISCUSSION THREAD

**Discussion by christiantricarico** [shibayan/appservice-acmebot#323] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8735590/118685874-c1287980-b803-11eb-93f9-a2d0791b97ef.png)

Hello,
maybe I miss something but I can't understand why I get this error message.
I used the acmebot function for another static website in the same resource group without problems.
Any advice?

**Comment by shibayan** [comment, marked as answer]
Have you added IP restrictions or AAD authentication so that the App Service cannot be accessed from outside?

**Reply by christiantricarico** [reply]
I finally fixed this issue.
It was caused by old extra IPv6 settings in the DNS settings.
I removed those IPv6 settings and it worked.
